### ISSUE TITLE
Doesnt handle property expansion in web.xml

### ISSUE THREAD

**Issue by nowheretobefound** [omnifaces/omnifaces#380]
OmniPartialViewContext doesnt pick up the login page from web.xml in this case :

```
<auth-method>${com.ec.frmw.authmethod:FORM}</auth-method>
```

```
<login-config>
        <auth-method>${com.ec.frmw.authmethod:FORM}</auth-method>
        <realm-name>default</realm-name>
        <form-login-config>
            <form-login-page>/xhtml/pages/login.jsf</form-login-page>
            <form-error-page>/xhtml/pages/login.jsf?s=1</form-error-page>
        </form-login-config>
    </login-config>
```

**Comment by BalusC** [comment]
I added a fallback where it ignores <auth-method>.

Fix is available in today's 2.6.3-SNAPSHOT. Can you give it a try?

**Comment by nowheretobefound** [comment]
What is the issue when redirecting to login.jsf and you end up on j_security_check after successful login ?
